### PR TITLE
Added minio service

### PR DESCRIPTION
**Pull request by luisdalmolin**
Added Minio service (https://github.com/minio/minio) for a local "s3" instance.

### CODE REVIEW

File: app/Services/Minio.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    protected $prompts = [
+        [
+            'shortname' => 'volume',
+            'prompt' => 'What is the Docker volume name?',
+            'default' => 'minio_data',
+        ],
+    ];
+
+    protected $dockerRunTemplate = '-p "$port":9000 \
+        -e MINIO_ACCESS_KEY=minio
```
Comment (mattstauffer) [inline app/Services/Minio.php:18]: I've never used Minio; is it likely people will want to change their access key and secret? If so, can you add those as prompts as well?

Comment (luisdalmolin) [reply app/Services/Minio.php:18]: Yeah, I guess it makes sense to put as prompts with default values. Minio is pretty cool for having a local S3 for development.

Comment (luisdalmolin) [reply app/Services/Minio.php]: Updated & pushed.

**Comment by mattstauffer** [comment]
Thanks!!

**Pull request merged**